tests/wp-unit/include/Core/Metas/Modules/DeleteTermMetaModuleTest.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace BulkWP\BulkDelete\Core\Metas\Modules;
+
+use BulkWP\Tests\WPCore\WPCoreUnitTestCase;
+
+/**
+ * Test Delete Term Meta Module.
+ *
+ * Tests \BulkWP\BulkDelete\Core\Metas\Modules\DeleteTermMetaModule
+ *
+ * @since 6.0.0
+ */
+class DeleteTermMetaModuleTest extends WPCoreUnitTestCase {
+
+	/**
+	 * The module that is getting tested.
+	 *
+	 * @var \BulkWP\BulkDelete\Core\Metas\Modules\DeleteTermMetaModule
+	 */
+	protected $module;
+
+	/**
+	 * Setup the Module.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->module = new DeleteTermMetaModule();
+	}
+
+	/**
+	 * Add to test delete default taxonomy term meta with equal value.
+	 */
+	public function test_that_term_meta_can_be_deleted_with_default_taxonomy_in_equal_value() {
```

Comment:
Rename to `test_that_term_meta_can_be_deleted_from_default_taxonomy_using_equal_operator.

Change the other function also to follow the correct format.

Reply:
Fixed.